### PR TITLE
Feature 5435 removal of python 2 7 related code

### PR DESCRIPTION
**Pull request by joeldierkes**
This PR removes the old Python2 related code.

### CODE REVIEW

File: lib/rucio/client/baseclient.py [hunk 1]
```diff
@@ -588,9 +586,6 @@ def __get_token_OIDC(self):
                 while count < 3:
                     # Python3 default
                     get_input = input
-                    # if Python version <= 2.7 use raw_input
-                    if sys.version_info[:2] <= (2, 7):
-                        get_input = raw_input  # noqa: F821 pylint: disable=undefined-variable
                     fetchcode = get_input()
```
Comment (rcarpa) [inline lib/rucio/client/baseclient.py]: we don't need to assign input to a different variable anymore

**Review by rcarpa — DISMISSED** [review]

File: lib/rucio/client/baseclient.py (hunk 1 above)
Comment (joeldierkes) [reply lib/rucio/client/baseclient.py]: Good catch! :+1:

**Review by mlassnig — APPROVED** [review]
:man_cook: oh yeah!

**Review by maany — APPROVED** [review]
Changes related to tests/ci are good to go

**Review by bari12 — APPROVED** [review]
Very nice 👍 Thanks!

**Pull request merged**